src/ApiNavigation.js

```diff
@@ -1843,14 +1868,15 @@ export class ApiNavigation extends AmfHelperMixin(LitElement) {
           ?compatibility="${this.compatibility}"
           tabindex="-1"
         >
-          <span class="icon" aria-label="Collapsed">${keyboardArrowDown}</span>
+          <span class="icon" aria-label="${ariaLabel}">${keyboardArrowDown}</span>
         </anypoint-icon-button>
       </div>
       <anypoint-collapse
         part="api-navigation-operation-collapse"
         class="operation-collapse"
         data-api-id="${item.id}"
         role="menu"
+        ?opened="${this._nestedEndpointsOpened}"
```

Comment:
There's a logic in the application code that programmatically opens/closes collapses on click. Does this interfere with it? When you click to close the collapse and the next render happens the `_nestedEndpointsOpened` still gonna be `true` and open the collapse.